week4/assignment/src/components/common/input.jsx

```diff
@@ -0,0 +1,34 @@
+import styled from "styled-components";
+
+const PerInput = ({ title, type, placeholder, value, onChange, disabled }) => {
```

Comment:
이렇게 받아오는 인자가 많을 땐 하나의 인자로 받아서 그 부분을 구조 분해 할당으로 바꿔주는 건 어때요?
```suggestion
const PerInput = ({ title, type, placeholder, value, onChange, disabled }) => {
CONST PerInput = (props) => {
const {title, type, placeholder, value, onChange, disabled}= props
```